src/coreclr/src/ToolBox/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -942,7 +939,6 @@ CorInfoInitClassResult MethodContext::repInitClass(CORINFO_FIELD_HANDLE   field,
     key.field       = (DWORDLONG)field;
     key.method      = (DWORDLONG)method;
     key.context     = (DWORDLONG)context;
-    key.speculative = (DWORD)speculative;
 
```

Comment:
I guess `speculative` field can be removed from `Agnostic_InitClass` struct

Reply:
It is removed already: https://github.com/dotnet/runtime/pull/38229/files#diff-0c48527a20aff4179c577db181af1963L66